### DISCUSSION TITLE
blog/how-to-use-document-tsx-in-next-13-version

### DISCUSSION THREAD

**Discussion by giscus** [yunjeoming/yuns-blog#41] [Comments]
# blog/how-to-use-document-tsx-in-next-13-version

12 버전 이하에서는 _document.tsx를 사용했는데, 13 버전부터는 어떻게 해야할까?

http://localhost:3000/blog/how-to-use-document-tsx-in-next-13-version

**Comment by yunjeoming** [comment]
test

**Reply by yunjeoming** [reply]
reply
